### PR TITLE
Fix dynamic island bug on iOS Edge

### PR DESCRIPTION
**Pull request by andy-mair-ons**
https://officefornationalstatistics.atlassian.net/browse/ONSDESYS-876

### What is the context of this PR?

- Adds viewport-fit="cover" to meta tag content fills edge-to-edge
- Adds the insets using CSS environment variables to control padding around hardware obstructions 

### How to review this PR

Check code works across browsers on iOS, android, mac and windows 

### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

**Review by SriHV — APPROVED** [review]
Tested in Android. Working as expected!

File: src/components/header/_header.scss [hunk 1]
```diff
@@ -367,6 +367,14 @@
             width: 100vw;
```
Comment (sveltifier) [inline src/components/header/_header.scss]: Try 100% or something and see if we can get rid of the extra special-case rule below

**Review by sveltifier — CHANGES_REQUESTED** [review]

File: src/components/header/_header.scss (hunk 1 above)
Comment (andy-mair-ons) [reply src/components/header/_header.scss]: Have converted the units to % and it should resolve the issue without the special case class

**Comment by sveltifier** [comment]
Updated PR title to make a better release note.

File: src/layout/_template.njk [hunk 2]
```diff
@@ -62,7 +62,7 @@
     <head>
         <meta charset="utf-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" viewport-fit="cover"/>
```
Comment (sveltifier) [inline src/layout/_template.njk]: I believe this is invalid HTML, and therefore ignored. 


<meta name="viewport" content="width=device-width, initial-scale=1" viewport-fit="cover"/>


<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>

If so this can & should be removed.

Comment (sveltifier) [reply src/layout/_template.njk]: <meta name="viewport" content="width=device-width, initial-scale=1" viewport-fit="cover"/>

    
    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>

**Review by sveltifier — CHANGES_REQUESTED** [review]

File: src/components/header/_header.scss [hunk 3]
```diff
@@ -364,7 +364,7 @@
         .ons-js-enabled & {
             position: relative;
             left: 50%;
-            width: 100vw;
+            width: 100%;
```
Comment (sveltifier) [inline src/components/header/_header.scss:366]: I think that 100% + left: 50% + translateX(-50%) is internally contradictory. When width: 100% equals the container's width, left: 50% shifts it off-centre and translateX(-50%) brings it back, net effect is zero movement. Why was this change necessary again? Chat if needed.

**Review by sveltifier — DISMISSED** [review]

**Comment by sveltifier** [comment]
Also, I tested on iOS, and the bug is fixed in Edge, and everything looked ok in Chrome and Safari too, so I think the CSS rule is working.

File: src/components/header/_header.scss (hunk 3 above)
Comment (andy-mair-ons) [reply src/components/header/_header.scss:366]: Had to change from vw to % to avoid the header overflowing the safe-area-inset paddings instead of having a custom edge case class. The left 50% and translateX(-50%) was there originally and is a common centring technique. Have tested without it across all the example headers and doesn't seem to be doing anything so have removed it
